semantics.json

```diff
@@ -51,6 +51,22 @@
     "label": "Enable autoplay",
     "description": "With autoplay the audio starts to play immediately. If autoplay is disabled the user presses a play button to start the audio."
   },
+  {
+    "name": "playAudio",
+    "type": "text",
+    "importance": "low",
+    "label": "Play audio (Readspeaker)",
+    "common": true,
+    "default": "Play audio"
+  },
+  {
+    "name": "pauseAudio",
+    "type": "text",
+    "importance": "low",
+    "label": "Pause audio (Readspeaker)",
+    "common": true,
+    "default": "Pause audio"
+  },
```

Comment:
These two new fields need to have default values set in the code (constructor), so it will be backward compatible

Reply:
Added